triggers/tests/BUILD.bazel

```diff
@@ -187,6 +187,7 @@ da_scala_library(
 [
     da_scala_test_suite(
         name = "trigger-failure-integration-tests" + suffix,
+        timeout = "moderate",
```

Comment:
Increased testing timeout (for `LoadTesting.scala`) as we appear to ow timeout in windows environments